cg/meta/workflow/mutant.py

```diff
@@ -0,0 +1,154 @@
+import datetime as dt
+import json
+import logging
+import shutil
+from pathlib import Path
+from typing import List
+
+from cg.constants import Pipeline
+from cg.meta.workflow.analysis import AnalysisAPI
+from cg.meta.workflow.fastq import MicrosaltFastqHandler
+from cg.models.workflow.mutant import MutantSampleConfig
+from cg.store import models
+from cg.utils import Process
+
+LOG = logging.getLogger(__name__)
+
+
+class MutantAnalysisAPI(AnalysisAPI):
+    def __init__(
+        self,
+        config: dict = None,
+        pipeline: Pipeline = Pipeline.SARS_COV_2,
+    ):
+        super().__init__(config=config, pipeline=pipeline)
+        self.root_dir = config["mutant"]["root"]
+
+    @property
+    def process(self) -> Process:
+        if not self._process:
+            self._process = Process(
+                binary=self.config["mutant"]["binary_path"],
+                environment=self.config["mutant"]["conda_env"],
+            )
+        return self._process
+
+    @property
+    def fastq_handler(self):
+        return MicrosaltFastqHandler
+
+    @property
+    def threshold_reads(self):
+        return False
```

Comment:
I don't really see the use case of this method.

Reply:
Its used in base class AnalysisAPI - this is a property general got all workflows
Threshold reads is used when grabbing cases to analyze. For some workflows its important that all samples passed sequencing QC before being analyzed (Balsamic, MIP need whole group of samples having adequate read count) while others can be ran anyway as the samples are independent in analysis (micro, fluffy, mutant)
I need to document better what it does though